notifications_utils/broadcast_areas/repo.py

```diff
@@ -0,0 +1,198 @@
+                simple_feature_geojson TEXT NOT NULL,
+
+                FOREIGN KEY (broadcast_area_library_id)
+                    REFERENCES broadcast_area_libraries(id),
+
+                FOREIGN KEY (broadcast_area_library_group_id)
+                    REFERENCES broadcast_area_library_groups(id)
+            )""")
+
+            conn.execute("""
+            CREATE INDEX broadcast_areas_broadcast_area_library_id
+            ON broadcast_areas (broadcast_area_library_id);
+            """)
+
+            conn.execute("""
+            CREATE INDEX broadcast_areas_broadcast_area_library_group_id
+            ON broadcast_areas (broadcast_area_library_group_id);
+            """)
+
+    def insert_broadcast_area_library(self, id, name, is_group):
+
+        q = """
+        INSERT INTO broadcast_area_libraries (id, name, is_group)
+        VALUES (?, ?, ?)
+        """
+
+        with self.conn() as conn:
+            conn.execute(q, (id, name, is_group))
+
+    def insert_broadcast_areas(self, areas):
+
+        q = """
+        INSERT INTO broadcast_areas (
+            id, name,
+            broadcast_area_library_id, broadcast_area_library_group_id,
+            feature_geojson, simple_feature_geojson
+        )
+        VALUES (?, ?, ?, ?, ?, ?)
+        """
+
+        with self.conn() as conn:
+            for id, name, area_id, group, feature, simple_feature in areas:
+                conn.execute(q, (
+                    id, name,
+                    area_id, group,
+                    geojson.dumps(feature), geojson.dumps(simple_feature),
+                ))
+
+    def query(self, sql, *args):
+        with self.conn() as conn:
+            cursor = conn.cursor()
+            cursor.execute(sql, (*args,))
+            return cursor.fetchall()
+
+    def get_libraries(self):
+        q = "SELECT id, name, is_group FROM broadcast_area_libraries"
+        results = self.query(q)
+        libraries = [(row[0], row[1], row[2]) for row in results]
+        return sorted(libraries)
+
+    def get_library_description(self, library_id):
+        q = """
+        WITH
+        areas AS (SELECT * FROM broadcast_areas
+                  WHERE broadcast_area_library_id = ?),
+        area_count AS (SELECT COUNT(*) AS c FROM areas),
+        subset_area_count AS (SELECT c - 4 FROM area_count),
+        some_area_names  AS (SELECT name FROM areas LIMIT 100),
+        some_shuffled_area_names AS (
+            SELECT name FROM some_area_names ORDER BY RANDOM()
+        ),
+        description_area_names AS (
+            SELECT name FROM some_shuffled_area_names LIMIT 4
+        ),
+        description_areas_joined AS (
+            SELECT GROUP_CONCAT(name, ", ") FROM description_area_names
+        )
+        SELECT
+        CASE (SELECT * FROM subset_area_count)
+        WHEN 0 THEN
+            (SELECT * FROM description_areas_joined)
+        ELSE
+            (SELECT * FROM description_areas_joined)
+            || ", "
```

Comment:
To match what the admin app does at the moment:
```suggestion
            || ", and "
```